WordPress/gutenberg
# [RNMobile] Update native BlockOutline component styles

## What?
Updates the native BlockOutline component and supporting styles.

Related:
* https://github.com/wordpress-mobile/gutenberg-mobile/pull/5833
* https://github.com/wordpress-mobile/WordPress-Android/pull/18575
* https://github.com/wordpress-mobile/WordPress-iOS/pull/20804


## Why?
As part of ongoing editor UX improvements, the solid border around outer blocks and dashed border around inner blocks are being removed. 

## How?
Update styles and references to `BlockOutline` component so that the main blue border around blocks is removed.

## Testing Instructions
1. Open a Post.
2. Tap a block.
3. Note that block outlines (solid or dashed) are not displayed.

## Screenshots

https://github.com/WordPress/gutenberg/assets/643285/f572becc-9912-404e-9f7d-246a6a7583aa
